Core/include/Acts/Propagator/SympyStepper.hpp

```diff
@@ -132,15 +133,8 @@ class SympyStepper {
     /// The geometry context
     std::reference_wrapper<const GeometryContext> geoContext;
 
-    /// @brief Storage of magnetic field and the sub steps during a RKN4 step
-    struct {
-      /// Magnetic field evaulations
-      Vector3 B_first, B_middle, B_last;
-      /// k_i of the RKN4 algorithm
-      Vector3 k1, k2, k3, k4;
-      /// k_i elements of the momenta
-      std::array<double, 4> kQoP{};
-    } stepData;
```

Comment:
Was this unused?

Reply:
Yes, I just stumbled across this while adding the statistics. The sympy stepper does not rely on these state variables.